content/en/guides/hosting/monitoring-usage/org_dashboard.md

```diff
@@ -6,92 +6,92 @@ menu:
-1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.
-
-    A user is _active_ if they:
-    - log in to W&B.
-    - view any page in the W&B App.
-    - log runs.
-    - use the SDK to track an experiment.
-    - interact with the W&B Server in any way.
+{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
+1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default and lists every user in the organization.
+2. To sort the list by user status, click the **Last Active** column header. Each user's status is one of the following:
+   * **Invite pending** – an invitation was sent but not yet accepted.
+   * **A timestamp** – the user accepted the invitation and has signed in at least once. The timestamp indicates the most recent activity.
+   * **–** – the user was previously active but has not been active in the last six months.
+   * **Deactivated** – an admin revoked the user's access.
+3. Hover over a user's **Last Active** field to see the date the user was added and their total active days.
+
+A user is _active_ if they:
+- sign in to W&B.
+- open any page in the W&B App.
+- log runs.
+- use the SDK to track an experiment.
+- interact with the W&B server in any way.
 {{% /tab %}}
 
 {{% tab header="Multi-tenant Cloud" value="saas" %}}
-1. Navigate to the [**Members** page](https://wandb.ai/account-settings/wandb/members/). This page lists all users, along with data about each user.
-1. To sort the list by user status, click the **Last Active** column label. Each user's status is one of the following:
-
-    * **Invite pending**: Admin has sent invite but user has not accepted invitation. 
-    * **Active**: User has accepted the invite and created an account.
-    * `-`: A hyphen indicates that the user has not yet been active within the organization.
-
-    A user is _active_ if they perform any auditable action scoped to the organization _after May 8, 2025_. For a full list, refer to [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in the Audit Logging page.
+1. Open the [**Members** page](https://wandb.ai/account-settings/wandb/members/). The table lists every user in your organization.
+2. Click the **Last Active** column header to sort by user status. Each user's status is one of the following:
+   * **Active** – the user accepted the invitation and has signed in at least once.
+   * **–** – the user has not yet been active within the organization.
 
+A user is _active_ if they perform any auditable action scoped to the organization _after May 8 2025_. For a full list, see [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in **Audit logging**.
```

Comment:
Suggestion: remove italicization for "after May 8 2025". It diminshes the impact of the other terms here that are italicized.